--- origsrc/gst-plugins-bad-0.11.99/ext/chromaprint/Makefile.am	2012-04-10 00:00:53.000000000 -0500
+++ src/gst-plugins-bad-0.11.99/ext/chromaprint/Makefile.am	2012-09-23 15:55:57.947648300 -0500
@@ -3,10 +3,10 @@ plugin_LTLIBRARIES = libgstchromaprint.l
 libgstchromaprint_la_SOURCES = gstchromaprint.c gstchromaprint.h
 
 libgstchromaprint_la_CFLAGS = $(GST_PLUGINS_BAD_CFLAGS) \
-	$(GST_PLUGINS_BASE_CFLAGS) $(GST_CFLAGS) \
+	$(GST_PLUGINS_BASE_CFLAGS) $(GST_BASE_CFLAGS) \
 	$(CHROMAPRINT_CFLAGS)
 libgstchromaprint_la_LIBADD = \
-	$(GST_PLUGINS_BASE_LIBS) -lgstaudio-$(GST_API_VERSION) $(GST_LIBS) \
+	$(GST_PLUGINS_BASE_LIBS) -lgstaudio-$(GST_API_VERSION) $(GST_BASE_LIBS) \
 	$(CHROMAPRINT_LIBS)
 libgstchromaprint_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS)
 libgstchromaprint_la_LIBTOOLFLAGS = --tag=disable-static
