--- origsrc/gst-plugins-bad-1.0.4/sys/d3dvideosink/d3dvideosink.c	2012-10-25 05:56:45.000000000 -0500
+++ src/gst-plugins-bad-1.0.4/sys/d3dvideosink/d3dvideosink.c	2012-12-18 22:51:17.088308200 -0600
@@ -2434,7 +2434,7 @@ plugin_init (GstPlugin * plugin)
 {
   /* PRIMARY: this is the best videosink to use on windows */
   if (!gst_element_register (plugin, ELEMENT_NAME,
-          GST_RANK_PRIMARY, GST_TYPE_D3DVIDEOSINK))
+          GST_RANK_NONE, GST_TYPE_D3DVIDEOSINK))
     return FALSE;
 
   return TRUE;
--- origsrc/gst-plugins-bad-1.0.4/sys/directdraw/gstdirectdrawplugin.c	2012-10-25 05:56:46.000000000 -0500
+++ src/gst-plugins-bad-1.0.4/sys/directdraw/gstdirectdrawplugin.c	2012-12-18 22:51:27.444900600 -0600
@@ -33,7 +33,7 @@
 static gboolean
 plugin_init (GstPlugin * plugin)
 {
-  if (!gst_element_register (plugin, "directdrawsink", GST_RANK_SECONDARY,
+  if (!gst_element_register (plugin, "directdrawsink", GST_RANK_NONE,
           GST_TYPE_DIRECTDRAW_SINK))
     return FALSE;
 
--- origsrc/gst-plugins-bad-1.0.4/sys/dshowvideosink/dshowvideosink.cpp	2012-10-25 05:56:46.000000000 -0500
+++ src/gst-plugins-bad-1.0.4/sys/dshowvideosink/dshowvideosink.cpp	2012-12-18 22:52:30.366499500 -0600
@@ -1932,7 +1932,7 @@ plugin_init (GstPlugin * plugin)
 {
   /* PRIMARY: this is the best videosink to use on windows */
   if (!gst_element_register (plugin, "dshowvideosink",
-          GST_RANK_SECONDARY, GST_TYPE_DSHOWVIDEOSINK))
+          GST_RANK_NONE, GST_TYPE_DSHOWVIDEOSINK))
     return FALSE;
 
   return TRUE;
